plugin/evm/atomic/params.go

```diff
@@ -0,0 +1,13 @@
+// (c) 2025 Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package atomic
+
+import "github.com/ava-labs/avalanchego/utils/units"
+
+const (
+	AvalancheAtomicTxFee = units.MilliAvax
+
+	// The base cost to charge per atomic transaction. Added in Apricot Phase 5.
+	AtomicTxBaseCost uint64 = 10_000
```

Comment:
nit perhaps a tiny tiny bit out of scope, but it would be nice to rename those whilst we're at it (same number of lines modified) to not have the atomic word in them given they're in the atomic package already. Something like

- `AvalancheAtomicTxFee` -> `AvalancheTxFee`
- `AtomicTxBaseCost` -> `TxBaseCost`